### PR TITLE
Adding oracle.can_query service check (DBMON-3620)

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do?

Adding `oracle.can_query` service check.

### Motivation

Compatibility with the old Oracle integration written in Python.

### Describe how to test/QA your changes

Check on monitors if `can_query` appears.

### CODE REVIEW

File: pkg/collector/corechecks/oracle-dbm/sender_util.go [hunk 1]
```diff
@@ -76,3 +77,22 @@ func sendMetric(c *Check, method metricType, metric string, value float64, tags
...
 }
+
+func sendServiceCheck(c *Check, service string, status servicecheck.ServiceCheckStatus, message string) {
+	sender, err := c.GetSender()
+	if err != nil {
+		log.Errorf("%s failed to get metric sender %s", err)
+		return
+	}
+
+	sender.ServiceCheck("oracle.can_query", status, c.dbHostname, c.tags, message)
```
Comment (lu-zhengda) [inline pkg/collector/corechecks/oracle-dbm/sender_util.go:88]: can we make `oracle.can_query` a constant something like
```go
const serviceCheckName = "oracle.can_query"
```

File: releasenotes/notes/oracle-can-query-c82d891454d06bfe.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    [oracle] Adding `oracle.can_query` service check.
```
Comment (urseberry) [inline releasenotes/notes/oracle-can-query-c82d891454d06bfe.yaml:11]:
```suggestion
    [oracle] Adds `oracle.can_query` service check.
```

**Review by urseberry — APPROVED** [review]

**Review by lu-zhengda — APPROVED** [review]

**Pull request merged**